docs/security/security.adoc

```diff
@@ -0,0 +1,90 @@
+= Cordova Security - Self Defence Checks
+
+As part of the Aerogear JS SDK module, there is a security self defence check package for cordova projects. This package offers multiple security checks that can be used to analyse the security of the device. This can be useful for apps to detect vulnerablities and react accordingly. For example, if the app detects security issue *X* then do *Y*, where *X* could be no lock screen is set and *Y* is show a system prompt to setup the lock screen.
```

Comment:
`As part of the Aerogear JS SDK module, there is a security self defence check package for cordova projects. `

Reply:
`This can be useful for apps to detect vulnerablities and react accordingly.`

Useful for **developers**